pkg/xen-tools/Dockerfile

```diff
@@ -26,10 +26,12 @@ ENV BUILD_PKGS \
     gettext iasl util-linux-dev ncurses-dev glib-dev \
     pixman-dev libaio-dev yajl-dev argp-standalone \
     linux-headers git patch texinfo tar libcap-ng-dev \
-    attr-dev flex bison cmake libusb-dev zstd-dev
+    attr-dev flex bison cmake libusb-dev zstd-dev \
+    python3 build-base ninja
+
 ENV BUILD_PKGS_arm64 dtc-dev
 
-ENV PKGS alpine-baselayout musl-utils bash libaio libbz2 glib pixman yajl keyutils libusb xz-libs libuuid sudo zstd-libs
+ENV PKGS alpine-baselayout libgcc musl-utils libaio libbz2 glib pixman keyutils libusb xz-libs libuuid zstd-libs
```

Comment:
@roja-zededa , I don't think `libgcc` is needed, it's is?

Reply:
PS: add libncursesw to PKGS should make get rid of the issue you described....

Reply:
Yes, we need libgcc, because I am getting this error: `/usr/lib/xen/bin/qemu-system-x86_64 
Error loading shared library libgcc_s.so.1: No such file or directory (needed by /usr/lib/xen/bin/qemu-system-x86_64)
`

Reply:
ok, got it.